backend/test/services/organization/organization_test.py

```diff
@@ -81,6 +82,17 @@ def test_create_organization_as_root(organization_svc_integration: OrganizationS
     assert created_organization.id is not None
 
 
+def test_create_organization_id_already_exists(
+    organization_svc_integration: OrganizationService,
+):
+    """Test that the root user is able to create new organizations when an extraneous ID is provided."""
+    created_organization = organization_svc_integration.create(
+        root, to_add_conflicting_id
+    )
```

Comment:
@KrisJordan when creating this test, I wanted to ask if we want to start standardizing all features to use a `Draft` model to specifically avoid issues with IDs when creating items, including for organizations, events, etc. Thoughts? For now, I added these tests to get to 100% coverage before such a refactor is done more widely.

Reply:
I think moving in this direction makes sense, but agree it's a larger refactoring that is orthogonal